miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/ProgressiveResponseDto.java

```diff
@@ -32,6 +33,14 @@ public long getToken() {
     return updates;
   }
 
+  public boolean isAgain() {
```

Comment:
I guess this is terminology from UNIX where `EAGAIN` means “repeat your request for more”. What do you want it to be called?